src/core_plugins/kibana/public/dashboard/listing/dashboard_listing.js

```diff
@@ -27,12 +27,12 @@ export function DashboardListingController($injector, $scope) {
   const sortableProperties = new SortableProperties([
     {
       name: 'title',
-      getValue: item => item.title,
+      getValue: item => item.title.toLowerCase(),
       isAscending: true,
     },
     {
       name: 'description',
-      getValue: item => item.description,
+      getValue: item => item.description.toLowerCase(),
```

Comment:
Good point.  It comes back as an empty string in practice, so no error. I could be extra safe and add a check for it, but the way the saved dashboards are filled with defaults, it will never be undefined, so that might be excessive checking.  Not to say that things in the future might not change and eventually this could be undefined....